tests/test_command_line_client/test_arf_to_hml.py

```diff
@@ -133,7 +133,7 @@ def test_get_questions_with_option_show_failed_rules():
     tests.any_test_help.get_questions_with_option_show_failed_rules(client)
 
 
-def test_if_not_installed_inquirer_with_option_show_failed_rules(capsys):
+def est_if_not_installed_inquirer_with_option_show_failed_rules(capsys):
```

Comment:
Why it's changed from test to est? This doesn't look like a clever way of disabling it. I think people would think that it's a typo.